apis/management.cattle.io/v3/rke_types.go

```diff
@@ -691,7 +728,7 @@ type RotateCertificates struct {
 
 type DNSConfig struct {
 	// DNS provider
-	Provider string `yaml:"provider" json:"provider,omitempty"`
+	Provider string `yaml:"provider" json:"provider,omitempty" norman:"default=kube-dns"`
```

Comment:
you should rebase with master, so you get this change: https://github.com/rancher/types/pull/865/files